transfers/transferargs.py

```diff
@@ -60,6 +60,9 @@ def get_parser(doc):
     parser.add_argument('--hide', action='store_true',
                         help='If set, hide the Transfers and SIPs in the '
                              'dashboard once they complete.')
+    parser.add_argument('--delete', action='store_true', help='If set, delete '
```

Comment:
@jrwdunham this file shouldn't have been committed. If I have my git flight rules correct then once this set of reviews is complete, I can rebase and squish everything and then reset this file to its original state as part of that process. Alternatively, I can checkout the previous version and place it into source control that way, resetting an original mistake.

Reply:
So you're saying that lines 63-65 here should not be part of this PR?

Reply:
Yup.